Adjust server config and databases sqls for 3.0

slashr00t opened the pull request:
I've prepared a fix for version 3.0. It looks like guys from Zabbix have changed package structure with sqls - now there's only a single file in an upper directory.
I also needed to adjust server config template. Maybe it's not the most elegant solution, but it works.

slashr00t commented:
I guess it's a big topic, most of the tests fail.

dj-wasabi commented:
Hi @slashr00t ,

Thank you for making time to create this PR, but I can't accept PR that fails the CI tests.
I'll leave this PR open if you want to make it work.

slashr00t commented:
Hi @dj-wasabi ,

I've fixed the code and it passed all the tests. I've also added a simple comment regarding this sqls placement difference.
If you have more suggestions please let me know.

dj-wasabi commented:
Thanks!
